static/js/sefaria/sefaria.js

```diff
@@ -1943,6 +1943,17 @@ _media: {},
           data["completions"][0].toLowerCase().replace('״','"') == query.slice(0, data["completions"][0].length).toLowerCase().replace('״','"') &&
           data["completions"][0] != query.slice(0, data["completions"][0].length))
   },
+  isGershayimVariant: function(query, data) {
+    function replace_gershayim_by_merkhaot(string) {
+        return string.replace('"', '״');
+    }
+    return (
+      !(data["is_ref"]) &&
+      data.completions &&
+      !data.completions.includes(query) &&
+      data.completions.map(x => replace_gershayim_by_merkhaot(x)).includes(replace_gershayim_by_merkhaot(query))
```

Comment:
I don't think `map()` will work in this case. It returns an array but you want a boolean. You're also checking `includes()` but don't you want to check for exact equality?

Reply:
we're replacing all the gershayim in all completions, and checking if the new array includes the quiry string (also after replacing)

Reply:
but using 'some' will be better

Reply:
ah! I misunderstood this. That makes sense.